packages/x-license-pro/src/utils/licenseScope.ts

```diff
@@ -0,0 +1 @@
+export type LicenseScope = 'pro' | 'premium';
```

Comment:
To be honest I was thinking to rework this to an `enum` because we are also using the strings themselves and it would be nice to avoid typos. It will also help if we have more licenses in the future. What do you think?